.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -30,3 +30,9 @@
     - [ ] N/A or can only be tested interactively
     - **OR**
     - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
+- **Tooling**
+    - [ ] I have considered the user experience from a tooling perspective and enumerated concerns in the summary. This includes:
```

Comment:
We should have two options here.  They don’t believe there is an impact, or check box that they believe there will be impact.

The maintainer need no follow up on the later.

Reply:
Added